### PR TITLE
Saleor 2962 warehouses in checkout

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I created test for warehouse in checkout

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-2962 Create test for warehouse in checkout](https://app.clickup.com/t/2549495/SALEOR-2962)

File: cypress/apiRequests/Checkout.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ export function createCheckout({
   }`;
   return cy
     .sendRequestWithQuery(mutation, auth)
-    .its("body.data.checkoutCreate.checkout");
+    .its("body.data.checkoutCreate");
 }
 export function addShippingMethod(checkoutId, shippingMethodId) {
```
Comment (eCzerniak) [inline cypress/apiRequests/Checkout.js:49]: Please add an empty line.

File: cypress/integration/allEnv/chcekout/warehouses.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  deleteShippingStartsWith
+} from "../../../utils/shippingUtils";
+
+describe("Warehouses in checkout", () => {
+  const startsWith = `CyWarehouseCheckout`;
+  let defaultChannel;
+  let usAddress;
+  let plAddress;
+  let warehouse;
+  it("should not be possible to buy product for country not listed in warehouse", () => {
```
Comment (eCzerniak) [inline cypress/integration/allEnv/chcekout/warehouses.js:21]: Please add an empty line.

File: cypress/utils/ordersUtils.js [hunk 3]
```diff
@@ -85,7 +85,6 @@ function assignVariantsToOrder(order, variantsList) {
     orderRequest.addProductToOrder(order.id, variantElement.id);
   });
 }
-
```
Comment (eCzerniak) [inline cypress/utils/ordersUtils.js:88]: Please add an empty line.

**Review by eCzerniak — COMMENTED** [review]
Please remove `.png` and `.mp4` files.

**Review by eCzerniak — APPROVED** [review]

**Review by orzechdev — APPROVED** [review]

File: cypress/integration/allEnv/chcekout/warehouses.js (hunk 2 above)
Comment (dominik-zeglen) [reply cypress/integration/allEnv/chcekout/warehouses.js:22]: Please don't act as a linter. Also why would you want to put an empty line here?

File: cypress/integration/allEnv/chcekout/warehouses.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
+import faker from "faker";
```
Comment (dominik-zeglen) [inline cypress/integration/allEnv/chcekout/warehouses.js:1]: There's a typo in path name

**Review by dominik-zeglen — APPROVED** [review]

File: cypress/integration/allEnv/chcekout/warehouses.js (hunk 2 above)
Comment (dominik-zeglen) [reply cypress/integration/allEnv/chcekout/warehouses.js:22]: Oh okay I get it

Comment (dominik-zeglen) [reply cypress/integration/allEnv/chcekout/warehouses.js:22]:
```suggestion
  
  it("should not be possible to buy product for country not listed in warehouse", () => {
```

**Pull request merged**